main.tf

```diff
@@ -17,6 +17,13 @@ resource "azurerm_key_vault" "kv" {
   enabled_for_template_deployment = true
   soft_delete_retention_days      = 90
 
+  network_acls {
```

Comment:
can you run `terraform fmt` on this file please? it looks unformatted

Reply:
terraform fmt has been executed